### PR TITLE
feat: added full stack observability quickstart

### PR DESCRIPTION
**Pull request by kidk**
# Summary

Migrating old quickstarts into new repository. This is the full stack observability dashboard showing information from all telemetry sources. 

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] Did you check that your alerts actually work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

**Review by jpvajda — APPROVED** [review]
@kidk I just scanned this PR as it's pretty large, if you tested it and it works OK, it's ready to merge unless you need a review from @polfliet

**Comment by kidk** [comment]
I've tested this one so it should work fine.

**Pull request merged**
